tools/topology/topology2/development/tplg-targets.cmake

```diff
@@ -428,4 +428,16 @@ SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-ptl-rt721-4ch-echoref.bin,DMIC0_ENHANCED_CAPTURE=true,\
 EFX_DMIC0_TDFB_PARAMS=line4_pass,EFX_DMIC0_DRC_PARAMS=dmic_default,\
 DEEPBUFFER_FW_DMA_MS=10,DEEP_BUF_SPK=true,SDW_JACK_ECHO_REF=true,SDW_SPK_ECHO_REF=true"
+
+# RT721 eval board with PCH-DMIC, sof_sdw_quirk_table with SOC_SDW_PCH_DMIC
+# Enable echo reference capture from jack and speaker output
+# Enable BT offload
+"cavs-sdw\;sof-ptl-rt721-4ch-bt-off-echoref\;PLATFORM=ptl,SDW_DMIC=1,NUM_SDW_AMP_LINKS=1,NUM_DMICS=4,\
```

Comment:
@udaymb How will the tplg file be used? The kernel will not look for the sof-ptl-rt721-4ch-bt-off-echoref.tplg. Will you rename it to sof-ptl-rt721-4ch.tplg in the file system?